tests/openssl_test.py

```diff
@@ -0,0 +1,15 @@
+from tests import LimitedTestCase, main
+
+class TestOpenSSL(LimitedTestCase):
```

Comment:
`import`, `class` and `main` are all redundant. Just `def test_import():` would work just fine. Would you care to remove them? Or I can do it.